hack/setup.sh

```diff
@@ -285,9 +460,32 @@ test_example() {
         exit 1
 }
 
+clean_up_git_repo() {
+      log "cleaning up git repo"
+
+      BRANCH="$(cat hack/git_entropy)"
+
+      add_ssh_token
+
+      pushd "$(mktemp -d)"
+            git clone "$GIT_WRITER_SSH_USER@$GIT_WRITER_SERVER:$GIT_WRITER_SERVER_PORT/$GIT_WRITER_PROJECT/$GIT_WRITER_REPOSITORY.git"
+            pushd "$GIT_WRITER_REPOSITORY"
+                  git push -d origin "$BRANCH"
+            popd
+      popd
+
+      log "done"
+      exit 0
+}
+
 delete_containers() {
         docker rm -f $REGISTRY_CONTAINER_NAME || true
         docker rm -f $KUBERNETES_CONTAINER_NAME || true
+        docker-compose -f hack/docker-compose.yaml down -v
+}
+
+delete_generated_repository_keys() {
+        rm hack/gitea-key.pub hack/gitea-key hack/gitea-server-public-key
```

Comment:
```suggestion
        rm hack/gitea-key.pub hack/gitea-key hack/gitea-server-public-key || true
```
so that it doesn't fail if you don't have the files